networkx/algorithms/coloring/greedy_coloring.py

```diff
@@ -258,75 +493,118 @@ def strategy_saturation_largest_first(G, colors):
     "connected_sequential": strategy_connected_sequential,
     "saturation_largest_first": strategy_saturation_largest_first,
     "DSATUR": strategy_saturation_largest_first,
+    "rlf": strategy_rlf,
 }
 
 
+@not_implemented_for("directed")
 @nx._dispatchable
-def greedy_color(G, strategy="largest_first", interchange=False):
-    """Color a graph using various strategies of greedy graph coloring.
-
-    Attempts to color a graph using as few colors as possible, where no
-    neighbors of a node can have same color as the node itself. The
-    given strategy determines the order in which nodes are colored.
-
-    The strategies are described in [1]_, and smallest-last is based on
-    [2]_.
+def greedy_color(G, strategy="largest_first", tabu=0, interchange=False):
```

Comment:
The new kwarg should be added at the end of the signature in order to prevent breaking changes in existing code which rely on positional arguments. See [the contributor guide](https://networkx.org/documentation/stable/developer/deprecations.html#policy) for further details if interested!